packages/config/config/devices/0x0131/psp02.json

```diff
@@ -61,6 +61,73 @@
 			"writeOnly": false,
 			"allowManualEntry": true
 		},
+		"8": {
+			"label": "PIR motion re-trigger min interval",
+			"unit": "8s ticks",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 127,
+			"defaultValue": 3,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"9": {
+			"label": "Light on time hysteresis",
+			"description": "time to keep the light on without motion",
+			"unit": "8s ticks",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 127,
+			"defaultValue": 4,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"10": {
+			"label": "battery level auto-report interval",
+			"unit": "ticks",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 127,
+			"defaultValue": 12,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"11": {
+			"label": "door state report interval",
+			"unit": "ticks",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 127,
+			"defaultValue": 12,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"12": {
+			"label": "illumination auto report interval",
+			"unit": "ticks",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 127,
+			"defaultValue": 12,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"13": {
+			"label": "temperature report interval",
```

Comment:
The casing is off on these labels (and the next ones)

Reply:
I didn't substantively review these files...its just a parse to remove descriptions...